test/unit/core/bridge.ts

```diff
@@ -1,6 +1,6 @@
 import * as fetchMock from 'fetch-mock';
 import * as sinon from 'sinon';
-import { AbstractBridge, BrowserBridge, CloudBridge, BridgeTimeout } from '../../../src/core/bridge';
+import { AbstractBridge, BridgeTimeout, BrowserBridge, CloudBridge } from '../../../src/core/bridge';
```

Comment:
🔥 reorder.

Reply:
^ So would this be: AbstractBridge, BrowserBridge, BridgeTimeout, CloudBridge? @jrmykolyn

Reply:
Dang never mind, i comes before o....